pages/common/vimdiff.md

```diff
@@ -1,16 +1,36 @@
 # vimdiff
 
-> Open up to four files in vim and show the differences between them.
-> See `vim` documentation for working with files and navigating within a window.
+> Open up two or more files in vim and show the differences between them.
+> More information: https://www.vim.org.
 
-- Open two files and show the differences (up to four files can be compared):
+- Open two files and show the differences:
 
 `vimdiff {{file1}} {{file2}}`
 
-- Open two files using a horizontal window split instead of the default vertical split:
+- Move the cursor to the window on the left|right:
 
-`vimdiff -o {{file1}} {{file2}}`
+`Ctrl + w {{h|l}}`
 
-- Move the cursor to the window on the left|right|up|down:
+- Jump to next difference:
 
-`Ctrl + w {{h|l|k|j}}`
+`[c`
+
+- Jump to previous difference:
+
+`]c`
+
+- Obtain difference from other window to current window:
```

Comment:
Perhaps this will read better?
```suggestion
- Copy differences from another window to the current window:
```

Reply:
This would actually be incorrect- only the highlighted difference is copied/obtained, and "another" is a bit confusing as this tldr assumes only two windows, so "other" would be a bit more clear than "another". My edit takes these points into consideration.

Reply:
Not sure I'm following you, but I'm not a vim user lol :P

Anyway, the point I was trying to get at here is that `Obtain` sounds very odd indeed here. Obtain generally means to "gain possession of something", which I somewhat doubt is the meaning you're after here. Is there a way we can reword this at all?

Reply:
Sorry I might not have been clear... I agree, "obtain" does sound a bit odd even though technically that word was [selected by the Vim author](http://vimdoc.sourceforge.net/htmldoc/diff.html#do) to match with the shortcut letter "o", so I have changed the description it mostly based on your suggestion since tldr should be intuitive for beginners. You can see that change in [my most recent update to the PR](https://github.com/tldr-pages/tldr/pull/4968/files).

Reply:
Ah, I see. I didn't realise that `Obtain` was a technical term. In that case, what about this?

```suggestion
- Obtain differences from other window in the current window:
```

Reply:
Unfortunately, that is still incorrect. This command only grabs one difference, not all of them.

Do you not like [my most recent commit wording](https://github.com/tldr-pages/tldr/pull/4968/files) on line 23?
`- Copy the highlighted difference from the other window to the current window:`

If you like the technical term, I'm happy to modify it to:
`- Obtain the highlighted difference from the other window to the current window:`

Reply:
Ah, I didn't notice that you'd changed it. Thanks!

I think it's fine with the new version.